modules/rosapps/applications/screensavers/mazescr/maze.c

```diff
@@ -684,7 +684,6 @@ LRESULT CALLBACK ScreenSaverProc(
 
             start_timer(hWnd, solve_delay);
             break;
-
```

Comment:
Please keep those because otherwise it makes hard to see at 1st sight where the previous case ends and where the new one starts.

Reply:
I did them, because the switch-cases in this func were inconsistent in regard whether to put an empty line in between the case-statements or not. see the whole func, then you'll see what I mean.